content/compose/multiple-compose-files/extends.md

```diff
@@ -52,30 +56,44 @@ services:
     volumes:
       - "/data"
 ```
-
-In this case, you get exactly the same result as if you wrote
+You get exactly the same result as if you wrote
 `docker-compose.yml` with the same `build`, `ports` and `volumes` configuration
 values defined directly under `web`.
 
-You can go further and define, or re-define, configuration locally in
-`compose.yaml`:
+To include the service `webapp` in the final project when extending services from another file, you need to explicitly include both services in your current Compose file. For example:
```

Comment:
Not sure I get it
Do you want the end result to have two services, with the same config (which doesn't make sense to me)?